OutOfSchool/OutOfSchool.WebApi/Enums/AccountStatus.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+namespace OutOfSchool.WebApi.Enums
+{
+    public enum AccountStatus
```

Comment:
`[JsonConverter(typeof(StringEnumConverter))]` attribute so swagger knows we have all enums serialized to string